package.json

```diff
@@ -71,7 +71,7 @@
     "prettier": "^1.13.7"
   },
   "scripts": {
-    "precommit": "lint-staged",
+    "prepush": "lint-staged",
```

Comment:
Does this work?  The linting only runs on the currently stagged files, but before a push, there will be no stagged files

Reply:
no it works, but it was a great dev experience :-)